website/src/components/Group/Inputs.tsx

```diff
@@ -1,46 +1,20 @@
 import { type ComponentProps, type FC, type PropsWithChildren } from 'react';
 
 import { listOfCountries } from './listOfCountries.ts';
+import type { NewGroup } from '../../types/backend.ts';
 
-const chooseCountry = 'Choose a country...';
+export const CountryInputNoOptionChosen = 'Choose a country...';
 
 const fieldMapping = {
-    groupName: {
-        id: 'group-name',
-        required: true,
-    },
-    institution: {
-        id: 'institution-name',
-        required: true,
-    },
-    contactEmail: {
-        id: 'email',
-        required: true,
-    },
-    country: {
-        id: 'country',
-        required: true,
-    },
-    line1: {
-        id: 'address-line-1',
-        required: true,
-    },
-    line2: {
-        id: 'address-line-2',
-        required: false,
-    },
-    city: {
-        id: 'city',
-        required: true,
-    },
-    state: {
-        id: 'state',
-        required: false,
-    },
-    postalCode: {
-        id: 'postal-code',
-        required: true,
-    },
+    groupName: 'group-name',
+    institution: 'institution-name',
```

Comment:
We can also change the id here - there's no reason the html id needs to be `institution-name` rather than `institution`
```suggestion
    institution: 'institution',
```
this can then be calculated from a kebabCase() js function

Reply:
I have changed the ids here to match the property names of the objects.